indico/MaKaC/conference.py

```diff
@@ -1882,17 +1882,13 @@ class Conference(CommonObjectBase, Locatable):
 
     fossilizes(IConferenceFossil, IConferenceMinimalFossil, IConferenceEventInfoFossil)
 
-    def __init__(self, creator, id="", creationDate = None, modificationDate = None):
+    def __init__(self, creator=None, id='', creationDate=None, modificationDate=None):
         """Class constructor. Initialise the class attributes to the default
             values.
            Params:
             confData -- (Dict) Contains the data the conference object has to
                 be initialised to.
         """
-        #IndexedObject.__init__(self)
-        if creator == None:
-            raise MaKaCError( _("A creator must be specified when creating a new Event"), _("Event"))
-        self.__creator = creator
```

Comment:
Yeah.. I guess I can get rid of it. the constructor isn't called in that many places anyway